gossip/hub/handlers.go

```diff
@@ -137,7 +137,7 @@ type PathHandlers map[string]AppHandler
 // an implementation of the http.Handler interface.
 type AppHandler struct {
 	info    *hubInfo
-	epPath  string
+	Name    string
```

Comment:
nit: reorder these so that the exported `Name` field is separated from the unexported fields.

Reply:
Done.